Lob-API-public.yml

```diff
@@ -125,6 +125,53 @@ tags:
       - [Java](https://github.com/lob/lob-java)
       - [Elixir](https://github.com/lob/lob-elixir)
 
+  - name: NCOA
+    x-traitTag: true
```

Comment:
I looked this up and see it is one of Redocly's custom extensions to the open-api spec. Based upon my initial review, I can see how it applies to the NCOA pattern here, but I want to understand it better long-term.

Reply:
https://redoc.ly/docs/api-reference-docs/vendor-extensions/redoc-supported-extensions/ I think the best explanation of its use is that it renders the tag description? I should actually make the other NCOA tags traitTags too...